.travis.yml

```diff
@@ -31,10 +31,32 @@ jobs:
       services: [docker]
       dist: xenial
       python: "3.7"
+      env:
+        - IMAGE_NAME=praekeltfoundation/malariaconnect
+        - REGISTRY_USER=praekeltorgdeploy
+        - secure: "NCYJda292/G3brt2E7x9wDLM2UsdcwSl0Q+6m+TtcGyJSCt7aSVW8wR2ts5+9nu2THG0h/aotKVCfUX2jycZdT5AX72pLpm48sosVP1X5Inwt16+FfaSZPwr7RvIQEQdfPbJUmUvZKCQ6EfGRAppcBDc46s+br2VKwbmZSHkMTua8qUwvayIYOoOFQSUYoWuIT3IiwxwSHTyHpOsKS7PkZ+boPWV4jcO2SIl2rfJzvI9Hx/GCqWNpngqCyP79DlDvFu+or/+D+HlvkfGhg4CmriewLGOHP+l3nxDsXDQO4OoYfZoxBs8e9TgeeuC+VhyFUqPf21OfOQVvh8M/aCCy/hmtPdRGFtcqwwK+sjIeC4EjRMKhS92XJeypdh+umlJNP1dMlxPiky8i/Rdl23cBX4tuVU5knTzMLB08S8eFoCUGYOKOUN2WO36J4b5GzbzVKoJRsWeNW1IoCad3t57+OrCI3db+HYCqxsuQS/xQsCxjmycYmtkAiRIMWdFZGXHqPg/IShTKlvigQzJj11D63Z4ujVA6qdPi2zdrUg6jETtjq5plmd9jUaWzLV72DkonckLzkAWvM6WpYPOlGN3qrWEWPTZaGpE2+1Rw1Z40aFaxb0nlE8/POOwMdJA+OZCAkUl6sNffCPWYdK1reSN4BxqqfLI7rEp222bGWXAaa8="
```

Comment:
If you haven't already regenerated this value, you can do so like:
```sh
travis encrypt 'REGISTRY_PASS="<password>"'
```
this `secure` value is specific to the repo it is used in.

Reply:
would this be me adding my own password for dockerhub?